### PR TITLE
Set appAccess optional chaining

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Make sure `appAccess` do not break apps each time we add a new one.

### CODE REVIEW

File: apps/backend-functions/src/orgs.ts [hunk 1]
```diff
@@ -70,17 +70,7 @@ async function notifyOnOrgMemberChanges(before: OrganizationDocument, after: Org
...
-      const accessChanged = (module: Module) => {
-        return after.appAccess[a][module] === true && (!before.appAccess[a] || before.appAccess[a][module] === false);
-      }
-      if (accessChanged('dashboard') || accessChanged('marketplace')) {
-        appAccessChanged = true;
-      }
-    }
-
-    return appAccessChanged;
+    return app.some(a => module.some(m => after.appAccess[a]?.[m] !== before.appAccess[a]?.[m]));
```
Comment (phpgeekfr) [inline apps/backend-functions/src/orgs.ts:73]: If I read the previous method well, `appAccessChanged` was set to true only if some app access changed from false to true. 
Your method will also set `appAccessChanged` to true if  app access changed from true to false

File: firebase.json [hunk 2]
```diff
@@ -41,6 +41,15 @@
...
     },
+    {
+      "target": "financiers",
+      "public": "dist/apps/festival",
+      "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
+      "rewrites": [{
+        "source": "**",
+        "destination": "/index.html"
+      }]
+    },
```
Comment (phpgeekfr) [inline firebase.json:52]: This is the missing config that was preventing us to deploy target ?

**Review by phpgeekfr — APPROVED** [review]

**Review by pldespaigne — APPROVED** [review]

File: apps/backend-functions/src/orgs.ts (hunk 1 above)
Comment (phpgeekfr) [reply apps/backend-functions/src/orgs.ts:73]: This will lead to unwantted email

File: firebase.json (hunk 2 above)
Comment (GrandSchtroumpf) [reply firebase.json:52]: I think. This part shouldn't have been in the PR, I didn't test it yet. Anyway, we need it.

File: apps/backend-functions/src/orgs.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/backend-functions/src/orgs.ts:73]: Yes, but the method is `AppAccessChanged` which implies in both ways. I wanted you to double check that.
I can move back to only false -> true if this is the expected behaviour

Comment (phpgeekfr) [reply apps/backend-functions/src/orgs.ts:73]: yes this is the expected behaviour. We only send email to orgAdmins when he is granted a new app access. Not when it is removed

**Pull request merged**